Stratis.Bitcoin.Tests/Wallet/WalletManagerTest.cs

```diff
@@ -1410,6 +3303,40 @@ private List<HdAddress> GenerateAddresses(int count)
             return addresses;
         }
 
+        private static (ExtKey ExtKey, string ExtPubKey) GenerateAccountKeys(Features.Wallet.Wallet wallet, string password, string keyPath)
+        {
+            var accountExtKey = new ExtKey(Key.Parse(wallet.EncryptedSeed, password, wallet.Network), wallet.ChainCode);
+            var accountExtendedPubKey = accountExtKey.Derive(new KeyPath(keyPath)).Neuter().ToString(wallet.Network);
+            return (accountExtKey, accountExtendedPubKey);
+        }
+
+        private static (PubKey PubKey, BitcoinPubKeyAddress Address) GenerateAddressKeys(Features.Wallet.Wallet wallet, string accountExtendedPubKey, string keyPath)
+        {
+            var addressPubKey = ExtPubKey.Parse(accountExtendedPubKey).Derive(new KeyPath(keyPath)).PubKey;
+            var address = addressPubKey.GetAddress(wallet.Network);
+
+            return (addressPubKey, address);
+        }
+
+        private static ConcurrentChain GenerateChainWithHeight(int blockAmount)
```

Comment:
Not strictly necessary but adding a Network as a parameter would make this more widely usable.